### PR TITLE
pythonPackages.markdownify: init at 0.10.1

### PR DESCRIPTION
**Pull request by milahu**

### CODE REVIEW

File: pkgs/development/python-modules/markdownify/default.nix [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+}:
+
+buildPythonPackage rec {
+  name = "markdownify";
+  version = "2021-11-17";
+
+  src = fetchFromGitHub {
+    owner = "matthewwithanm";
+    repo = "python-markdownify";
+    rev = "478b1c7e1364075ab8f6c8c3e00731cb06415793";
```
Comment (fabaff) [inline pkgs/development/python-modules/markdownify/default.nix:16]:
```suggestion
    rev = version;
```

File: pkgs/development/python-modules/markdownify/default.nix [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    sha256 = "rvYqcGZqOvWa0SqDUtG49oZ9tR+TK7wcmBWeTCZh7dU=";
+  };
+
+  propagatedBuildInputs = [ beautifulsoup4 ];
```
Comment (fabaff) [inline pkgs/development/python-modules/markdownify/default.nix:20]: Upstream also mention [`six`](https://github.com/matthewwithanm/python-markdownify/blob/develop/setup.py#L79).

**Comment by milahu** [comment]
@fabaff thanks for your review : )

File: pkgs/development/python-modules/markdownify/default.nix [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  src = fetchFromGitHub {
+    owner = "matthewwithanm";
+    repo = "python-markdownify";
+    rev = version;
+    sha256 = "rvYqcGZqOvWa0SqDUtG49oZ9tR+TK7wcmBWeTCZh7dU=";
+  };
+
+  propagatedBuildInputs = [ beautifulsoup4 six ];
+
+  nativeBuildInputs =  [ flake8 ];
```
[the remark this answers is not in the document]
Comment (milahu) [reply pkgs/development/python-modules/markdownify/default.nix:23]: oops. note to self: also do a clean build

patch is in https://github.com/matthewwithanm/python-markdownify/pull/57

**Review by jonringer — COMMENTED** [review]
PR should be a single commit

Please follow [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md) and [manual#submitting-changes-making-patches ](https://nixos.org/nixpkgs/manual/#submitting-changes-making-patches) and squash the fix-up commits.

 `git rebase -i` is a powerful command which achieves this, I created a small video demonstrating it's use [here](https://www.youtube.com/watch?v=OR8Q-4PLalc). A more indepth text tutorial can be found [here](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase)

**Comment by jonringer** [comment]
When upstreaming bugs, I would try to confine the language to the domain. What I mean by this is that nixos is not really relevant to the fact that there's a newer version of flake, and that new version is still compatible with the tests.

**Comment by onny** [comment]
Thank you for your contribution, unfortunately this module was already merged into master in the meantime

**Pull request closed without merging**